test/EmptyApp.FunctionalTests/FunctionalTest/TelemetryModuleWorkingEmptyAppTests.cs

```diff
@@ -7,19 +7,20 @@ public class TelemetryModuleWorkingEmptyAppTests : TelemetryTestsBase
     {
         private const string assemblyName = "EmptyApp.FunctionalTests";
 
-#if NET451
-
         [Fact]
         public void TestBasicDependencyPropertiesAfterRequestingBasicPage()
         {
+#if NET451
```

Comment:
Interesting, well, maybe add a comment to that effect and then go ahead and complete the PR.